cmd/ebrelayer/txs/relayToCosmos.go

```diff
@@ -9,33 +9,35 @@ package txs
 // ------------------------------------------------------------
 
 import (
-	amino "github.com/tendermint/go-amino"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	authtxb "github.com/cosmos/cosmos-sdk/x/auth/types"
-
 	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
-
+	authtxb "github.com/cosmos/cosmos-sdk/x/auth/types"
```

Comment:
nit, this should be 
```suggestion
	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
```

Reply:
good find https://github.com/cosmos/peggy/commit/52fdedf6f0b4e4b1c7cf822ab3519dabba3afa17